content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,218 @@
+----------------------
+
+The following conditions are provided as an example of a good but not comprehensive set of rules for
+finding good leads:
+
+#. Lead creation date
+#. Sales team location
+#. Contains a phone number
+#. An active state is set
+#. Referred by appointment
+#. Source is livechat
+#. Notes are set
+#. Tagged with a size from 10-20 or 20-50
+#. Email does not contain bad addresses
+#. An active salesperson is set
+
+Any number of conditions can be added, removed, or modified to best fit the desired information in
+the report.
+
+Add lead creation date
+~~~~~~~~~~~~~~~~~~~~~~
+
+Click the first field on the left under :guilabel:`Match any of the following rules:` with the value
+:guilabel:`Country` in it. Type :guilabel:`Date` in the :guilabel:`Search...` bar, or scroll to
+search through the list to locate it. Then, click the next field and select :guilabel:`>=` from the
+drop-down menu. This operator will include only values greater than or equal to the value in the
+rightmost field.
+
+The rightmost field should contain the earliest date leads are selected from. For example,
+setting `01/01/2024 00:00:05` will include only leads created from, and including, the first day of
+2024.
+
+.. image:: lead_distribution_report/created-on.png
+   :align: center
+   :alt: Add a Created On rule for the start of the year onward.
+
+Add sales team location
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule and another horizontal set of fields will appear. Click
+the first field for the new rule, and set it to :guilabel:`Sales Team`. Then, click the next
+field and select :guilabel:`contains` from the drop-down menu. Selecting this operator will filter
+for any records that contain the words in the rightmost field.
+
+In this third field, enter the name of the desired sales teams. For example, setting `us direct sf
+northam` will include only sales teams assigned to the U.S., direct, San Francisco, and North
+America.
+
+.. image:: lead_distribution_report/sales-team-location.png
+   :align: center
+   :alt: Use Sales Team to filter the location the lead is associated with.
+
+Add phone number
+~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, click the next field and select :guilabel:`is set` from the drop-down menu.
+Selecting this operator will filter only for records that have a phone number associated with the
+lead.
+
+.. image:: lead_distribution_report/phone-set.png
+   :align: center
+   :alt: Report only leads with an associated phone number with the Phone field.
+
+Add active status
+~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of :guilabel:`Phone is set` to add a new rule that
+branches from the rules above.
+
+Two horizontal sets of fields will appear below a line showing
+:guilabel:`all of:` with a :icon:`fa-caret-down` down arrow. Click the down arrow, then select
+:guilabel:`any` from the list.
```

Comment:
When I do this the branch goes away, and falls in line with the other Rule lines above it. Is this what is intended? If so, I'm a bit unclear why the Branch needs to be created, and not just another rule. Maybe it can stay on the default 'all of' setting to keep the Branch look/functionality looking more unique? What do you think?